### PR TITLE
theme: make doc url dynamic

### PR DESCRIPTION
**Pull request by gautesl**
Changed the static `"/nRF_Connect_SDK/doc/"` string to instead be fetched from the URL pathname.

### CODE REVIEW

File: sphinx_ncs_theme/static/js/ncs.js [hunk 1]
```diff
@@ -19,7 +18,9 @@ function NCS () {
     if (LOCALHOST_RE.test(host)) {
       this.url_prefix = "/";
     } else {
-      this.url_prefix = NCS_PATH_PREFIX;
+      let path = window.location.pathname;
+      let prefix_end_index = path.indexOf("/doc/") + 5;
```
Comment (gmarull) [inline sphinx_ncs_theme/static/js/ncs.js]: Maybe a comment to explain why 5?

Comment (gautesl) [reply sphinx_ncs_theme/static/js/ncs.js]: I thought it implicit due to the "/doc/" string appears just before, but I can add one

Comment (gmarull) [reply sphinx_ncs_theme/static/js/ncs.js]: then maybe do `"/doc/".length` ?

Comment (gautesl) [reply sphinx_ncs_theme/static/js/ncs.js]: Fixed, thanks!

**Review by gmarull — APPROVED** [review]

**Pull request merged**